terasoluna-gfw-libraries-parent/pom.xml

```diff
@@ -0,0 +1,89 @@
+<project xmlns="http://maven.apache.org/POM/4.0.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+    xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
+    <modelVersion>4.0.0</modelVersion>
+    <artifactId>terasoluna-gfw-libraries-parent</artifactId>

```

Comment:
- [ ] I suggest `terasoluna-gfw-common-libraries`.

and I think the following directory structure is proper.
```
terasoluna-gfw
  - terasoluna-gfw-common-libraries
    - terasoluna-gfw-common ...and others...
  - terasoluna-gfw-dependencies
    - terasoluna-gfw-common-dependencies ...and others...
```